scripts/source-maintenance.sh

```diff
@@ -35,6 +35,14 @@ while [ $# -ge 1 ]; do
         KeepGoingDirective=$1
         shift
         ;;
+    --skip-astyle-version-check)
```

Comment:
I suggest to follow ./configure conventions. This suggestion also removes the ambiguity -- some could say that this option does not disable the check.

```suggestion
    --disable-astyle-version-enforcement)
```

Reply:
I like the idea but perhapse --enable-astyle=VERSION would be better. Either disable it by default or have the approved version as default. That way allows new versions to be tested nicely while also preventing the problems associated with any randomly installed version altering the code.

Reply:
okay, switching to --enable-astyle=VERSION ; I like the model better